client/src/Components/relatedItems/relatedCarousel/relatedCard/cardComponents/image.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Button from './actionButton.jsx';
+
+function Image({ images, handlesComparePopup }) {
+  return (
+    <div className="image-box">
+      <img src={images}></img>
+      <Button className="compare-button" handlesComparePopup={handlesComparePopup}>*Placeholder*</Button>
```

Comment:
Should 'handlesComparePopup={handlesComparePopup}' be 'onClick={handlesComparePopup}'?

Reply:
sorry i confused myself on here too by making another file just for the button. so im just passing the handlesComparePopup function down to the button. But I do have syntax errors here though.